### ISSUE TITLE
get error error when (alignment: Alignment.bottomCenter)

### ISSUE THREAD

**Issue by behzad-tg** [payam-zahedi/toastification#108]
this error happend when alignment set to bottomCenter, other direction is fine

E/flutter (12475): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: 'package:flutter/src/widgets/overlay.dart': Failed assertion: line 207 pos 12: '_overlay != null': is not true.
E/flutter (12475): #0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:51:61)
E/flutter (12475): #1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)
E/flutter (12475): #2      OverlayEntry.remove (package:flutter/src/widgets/overlay.dart:207:12)
E/flutter (12475): #3      ToastificationManager.dismiss.<anonymous closure> (package:toastification/src/core/toastification_manager.dart:147:28)
E/flutter (12475): #4      new Future.delayed.<anonymous closure> (dart:async/future.dart:423:39)
E/flutter (12475): #5      Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
E/flutter (12475): #6      _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
E/flutter (12475): #7      _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
E/flutter (12475): #8      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
E/flutter (12475):

**Comment by payam-zahedi** [comment]
@behzad-tg Hey behzad 

Thanks for opening this issue

Can you give me more information 

What is your Toastification version?

What is your platform that you are running the app?

**Comment by behzad-tg** [comment]
toastification: ^2.0.0
platfrom: Android

**Comment by edmonddc** [comment]
This is what I got:

```
../dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 297:3  throw_
../packages/flutter/src/animation/animation_controller.dart 823:9               <fn>
../packages/flutter/src/animation/animation_controller.dart 833:14              dispose
../packages/flutter/src/widgets/animated_scroll_view.dart 1198:55               <fn>
../dart-sdk/lib/async/zone.dart 1661:54                                         runUnary
../dart-sdk/lib/async/future_impl.dart 162:18                                   handleValue
../dart-sdk/lib/async/future_impl.dart 838:44                                   handleValueCallback
../dart-sdk/lib/async/future_impl.dart 867:13                                   _propagateToListeners
../dart-sdk/lib/async/future_impl.dart 643:5                                    [_completeWithValue]
../dart-sdk/lib/async/future_impl.dart 713:7                                    callback
../dart-sdk/lib/async/schedule_microtask.dart 40:11                             _microtaskLoop
../dart-sdk/lib/async/schedule_microtask.dart 49:5                              _startMicrotaskLoop
../dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 181:7           <fn>

```

It's triggering the debugger.

This is my code:

```
                                                    toastification.show(                                                      
                                                        context: context,
                                                        icon: const Icon(
                                                            Icons.check),
                                                        title:
                                                            const Text("Saved"),
                                                        alignment: Alignment
                                                            .bottomCenter,
                                                        primaryColor:
                                                            Colors.white,
                                                        showProgressBar: false,
                                                        backgroundColor:
                                                            Colors.green,
                                                        foregroundColor:
                                                            Colors.white);

```

**Comment by AntonioAEMartins** [comment]
Hi @payam-zahedi, @behzad-tg,

I believe the bug is not caused by Toastification. I tested using bottomCenter, and the toast responds correctly.

@edmonddc, this error was due to other issues, which are described and have already been corrected in this pull request: [#133](https://github.com/payam-zahedi/toastification/pull/133).

@payam-zahedi I think that we can close this issue.

**Comment by payam-zahedi** [comment]
Thank you @AntonioAEMartins 

I will close this issue for now.

But @behzad-tg, @edmonddc  feel free to reopen it if you still have the same issue
